### PR TITLE
Add mulled-v2-5b53309c4f8449fec3814df567dbf53f97b59e8e:e28bbdedb596096aed9625f6662b8568960f36ca for Trinotate 3.2.2

### PR DESCRIPTION
**Pull request by natefoo**

### CODE REVIEW

**Comment by mvdbeek** [comment]
Gonna close this, I think this should be picked up automatically by the bot (which was suspended due to inactivity ...)

File: combinations/mulled-v2-5b53309c4f8449fec3814df567dbf53f97b59e8e:e28bbdedb596096aed9625f6662b8568960f36ca.tsv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#targets	base_image	image_build
+trinotate=3.2.2,gnu-wget=1.18	bgruening/busybox-bash:0.1	0
```
Comment (mvdbeek) [inline combinations/mulled-v2-5b53309c4f8449fec3814df567dbf53f97b59e8e:e28bbdedb596096aed9625f6662b8568960f36ca.tsv:2]:
```suggestion
trinotate=3.2.2,gnu-wget=1.18	quay.io/bioconda/base-glibc-busybox-bash:latest	0
```

**Comment by natefoo** [comment]
I don't see a merge build for this and it's not in quay.io as far as I can tell.

**Comment by natefoo** [comment]
My mistake, it's there, I forgot to look for the image with the `-0` appended to the tag.

**Pull request merged**